front/lib/resources/storage/models/labs_transcripts.ts

```diff
@@ -69,6 +73,10 @@ LabsTranscriptsConfigurationModel.init(
       allowNull: false,
       defaultValue: false,
     },
+    gongApiKey: {
```

Comment:
Nah we don't, this would mean adding a new provider in the actual product code - not sure it's worth it, as we want to move away from Gong anyways?

Reply:
I thiiink that's desirable as it will make the code simpler and will reduce the surface of the refactor of Nango :+1:

Reply:
Creating a custom flow will actually create more work than less in a sense.

But this is obviously different (API key vs oauth)

My worry is that API key will lose many users esp in the context of a "personal use case" ?

More meta thinking: is this personal or is this for all calls here? If the later that's a divergence of the mental model of the current product?

Reply:
True, I was going to go for finding the gong userId from the api and get the transcripts only for that id but this could be a rabbithole. I'll check switching to oAuth, putting this PR back to draft